### PR TITLE
Added tooltip on copy button

### PR DESCRIPTION
**Pull request by Chakravarthy7102**

### CODE REVIEW

**Review by friedger — COMMENTED** [review]
Sorry for the late comment. I had pending comments and thought I submitted.

I like the copied message. However, I don't understand the hideOnClick code

File: src/components/tooltip.tsx [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 
 import { ExplainerIcon } from './icons/explainer';
 
-interface TooltipProps extends BoxProps {
+interface TooltipProps extends BoxProps, Partial<Pick<TippyProps, 'hideOnClick'>> {
   text: string;
 }
 
-export const Tooltip: FC<TooltipProps> = ({ children, text, ...props }) => {
+export const Tooltip: React.FC<TooltipProps> = ({ children, text, hideOnClick, ...props }) => {
```
[the remark this answers is not in the document]
Comment (Chakravarthy7102) [reply src/components/tooltip.tsx:12]: Hi @friedger, When we click on the copy button the tooltip stays for a sec and disappers and this behaviour can be controlled by [tippy docs](https://atomiks.github.io/tippyjs/v6/all-props/#hideonclick), So it just makes sure that after clicking on the copy button the tooltip wont close instantly.

**Comment by friedger** [comment]
LGTM 
@kyranjamie ?

**Comment by markmhx** [comment]
@Chakravarthy7102 did you delete the head repo here intentionally? We haven't merged in these changes yet

**Pull request closed without merging**